tools/scalajsenv.js

```diff
@@ -761,21 +777,24 @@ ScalaJS.TypeData.prototype.initClass = function(
 initClass(
 //!endif
     internalNameObj, isInterface, fullName,
-    ancestors, isRawJSType, parentData, isInstance, isArrayOf) {
+    ancestors, typeTag, isRawJSType, parentData, isInstance, isArrayOf) {
   const internalName = ScalaJS.propertyName(internalNameObj);
```

Comment:
No it should be the same commit, since your commit turns this into dead code, so it should also remove it.